### PR TITLE
sidenav

### PR DESCRIPTION
**Pull request by maria-pigazzi**
Not perfect sidenav in all pages apart from home (normal nav) and login/sign up (no nav).
Code possibly to refactor, could clash with previously merged css.
Please leave comments :)

### CODE REVIEW

**Comment by maria-pigazzi** [comment]
Also missing link's paths, waiting for all pages to be there to add them.

**Review by dumboluzz — COMMENTED** [review]
This created a huge merge conflict with @sophiecs333 's code. Maybe you too should resolve the conflict together.

File: app/views/shipments/_admin_index.html.erb [hunk 1]
```diff
@@ -1,71 +1,58 @@
...
+    <div class="row d-flex text-align-start mb-2">
+      <h1>Shipments<h1>
+    </div>
+    <div class="d-flex justify-content-evenly">
+      <h5 class="btn-tabs">All</h5>
+      <h5 class="btn-tabs">Active</h5>
+      <h5 class="btn-tabs">Delivered</h5>
+      <h5 class="btn-tabs">Check Finn others</h5>
+    </div>
+    <br>
```
[the remark this answers is not in the document]
Comment (maria-pigazzi) [reply app/views/shipments/_admin_index.html.erb:12]: Yes, the sidenav is child of a flex box parent that can be found in the application.html of this branch in theory

[the remark this answers is not in the document]
Comment (maria-pigazzi) [reply app/views/shipments/_admin_index.html.erb:12]: So I removed it from the index page html (that is the second child of the flex element, as yield in application.html)

[the remark this answers is not in the document]
Comment (dumboluzz) [reply app/views/shipments/_admin_index.html.erb:12]: What you you could do is following:



a flexbox with 2 children: Your nav and a div with flex-grow. Inside of that div you yield the page content

[the remark this answers is not in the document]
Comment (maria-pigazzi) [reply app/views/shipments/_admin_index.html.erb:12]: sidenav

[the remark this answers is not in the document]
Comment (maria-pigazzi) [reply app/views/shipments/_admin_index.html.erb:12]: I thought I was doing that in this passage

**Review by dumboluzz — APPROVED** [review]
The shipment index page still looks broken on my pc. But we gotta fix that after we merge.
Also on mobile the sidenav takes up 50% of the screen.

**Pull request merged**